docs/timeseries_pipeline_elements.md

```diff
@@ -0,0 +1,367 @@
+ 2020-01-20 00:00:00+00 | -1.23
+ 2020-01-21 00:00:00+00 | -0.29
+ 2020-01-22 00:00:00+00 | -0.37
+ 2020-01-23 00:00:00+00 |  1.48
+ 2020-01-24 00:00:00+00 | -0.52
+ 2020-01-25 00:00:00+00 |  1.34
+ 2020-01-26 00:00:00+00 | -0.95
+ 2020-01-27 00:00:00+00 | -0.65
+ 2020-01-28 00:00:00+00 | -0.42
+ 2020-01-29 00:00:00+00 |  1.42
+ 2020-01-30 00:00:00+00 | -0.66
+```
+
+Or even run one of our device's deltas through lttb to get a nice graphable set of points:
+```SQL
+SELECT (deltas |> toolkit_experimental.lttb(10))::TEXT FROM daily_delta where device = 7;
+```
+```output
+                                                                            text                                                                                                                                                                                                                                                                                               
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+ [{"ts":"2020-01-02 00:00:00+00","val":0.5555802022457712},{"ts":"2020-01-05 00:00:00+00","val":-1.4688929826077484},{"ts":"2020-01-08 00:00:00+00","val":2.416048415988122},{"ts":"2020-01-09 00:00:00+00","val":-3.0046993833401174},{"ts":"2020-01-14 00:00:00+00","val":0.22758839123397223},{"ts":"2020-01-17 00:00:00+00","val":-2.1256090660578124},{"ts":"2020-01-19 00:00:00+00","val":1.2272792346941657},{"ts":"2020-01-25 00:00:00+00","val":-3.1053238977555324},{"ts":"2020-01-26 00:00:00+00","val":1.2629388469236815},{"ts":"2020-01-30 00:00:00+00","val":-0.7042437967407409}]
+```
+
+## Current Pipeline Elements(A-Z) <a id="timeseries-pipeline-elements"></a>
+
+As of the current timescale release, these elements are all [experimental](/docs/README.md#tag-notes).
+
+
+> - [delta](#timeseries_pipeline_delta)
+> - [fill_holes](#timeseries_pipeline_fill_holes)
+> - [lttb](#timeseries_pipeline_lttb)
+> - [resample_to_rate](#timeseries_pipeline_resample_to_rate)
+> - [sort](#sort)
+
+
+---
+
+## **delta** <a id="timeseries_pipeline_delta"></a>
+```SQL ,ignore
+delta(
+) RETURNS TimeseriesPipelineElement
+```
+
+This element will return a new timeseries where each point is the difference between the current and preceeding value in the input timeseries.  The new series will be one point shorter as it will not have a preceding value to return a delta for the first point.
+
+### Required Arguments <a id="timeseries_pipeline_delta-arguments"></a>
```

Comment:
Just FYI, when you move these over to our docs, we use the form of

`[](#timeseries_pipeline_delta-arguments)` for anchor links because we have a few different ways of creating the in-page links so this allows consistency.